server/routes/components.ts

```diff
@@ -90,11 +90,15 @@ export default function routes({ serviceCatalogueService, redisService }: Servic
 
     const component = await serviceCatalogueService.getComponent(componentName)
     const environments = component.environments?.filter(environment => environment.name === environmentName)
-
+    let activeAgencies = ''
+    if (environments.length > 0) {
+      activeAgencies = formatActiveAgencies(environments[0].active_agencies as Array<string>)
+    }
```

Comment:
Can make this a ternary, sorry it's fine as is, I just try to avoid simple `if` and `let` if I can 😄 

`const activeAgencies = environments.length === 0 ? '' : formatActiveAgencies(environments[0].active_agencies as Array<string>)`

Reply:
did comment then read back - 👍🏻